app/views/ban_mailer/ban_email.html.erb

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
I've left some minor feedback about the email content [on the issue](https://github.com/TheOdinProject/theodinproject/issues/4666).